connection.go

```diff
@@ -207,25 +210,27 @@ type Opts struct {
 // Address could be specified in following ways:
 //
 // TCP connections:
-// - tcp://192.168.1.1:3013
-// - tcp://my.host:3013
-// - tcp:192.168.1.1:3013
-// - tcp:my.host:3013
-// - 192.168.1.1:3013
-// - my.host:3013
+//  - tcp://192.168.1.1:3013
+//  - tcp://my.host:3013
+//  - tcp:192.168.1.1:3013
+//  - tcp:my.host:3013
+//  - 192.168.1.1:3013
+//  - my.host:3013
+//
 // Unix socket:
-// - unix:///abs/path/tnt.sock
-// - unix:path/tnt.sock
-// - /abs/path/tnt.sock  - first '/' indicates unix socket
-// - ./rel/path/tnt.sock - first '.' indicates unix socket
-// - unix/:path/tnt.sock  - 'unix/' acts as a "host" and "/path..." as a port
+//  - unix:///abs/path/tnt.sock
+//  - unix:path/tnt.sock
+//  - /abs/path/tnt.sock  - first '/' indicates unix socket
+//  - ./rel/path/tnt.sock - first '.' indicates unix socket
+//  - unix/:path/tnt.sock - 'unix/' acts as a "host" and "/path..." as a port
 //
 // Note:
 //
 // - If opts.Reconnect is zero (default), then connection either already connected
 // or error is returned.
 //
-// - If opts.Reconnect is non-zero, then error will be returned only if authorization// fails. But if Tarantool is not reachable, then it will attempt to reconnect later
+// - If opts.Reconnect is non-zero, then error will be returned only if authorization
+// fails. But if Tarantool is not reachable, then it will attempt to reconnect later
 // and will not end attempts on authorization failures.
 func Connect(addr string, opts Opts) (conn *Connection, err error) {
```

Comment:
There are a lot there to proofread too

![image](https://user-images.githubusercontent.com/20455996/161789521-4dc750ac-d9f7-4e99-a19b-469d90e78406.png)


http://127.0.0.1:6060/pkg/github.com/tarantool/go-tarantool/#Connection.Addr